library/hdfs-library/src/main/java/uk/gov/gchq/gaffer/hdfs/operation/handler/job/tool/SampleDataAndCreateSplitsFileTool.java

```diff
@@ -155,7 +155,7 @@ private void writeSplits(final FileSystem fs, final Path resultsFile, final long
         try (final SequenceFile.Reader reader = new SequenceFile.Reader(fs.getConf(), Reader.file(resultsFile));
              final PrintStream splitsWriter = new PrintStream(
                      new BufferedOutputStream(fs.create(new Path(operation.getSplitsFilePath()), true)),
-                     false, CommonConstants.UTF_8)
+                     false, String.valueOf(StandardCharsets.UTF_8))
```

Comment:
Once we move to using Java 11 [we can use the Charset directly](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/PrintStream.html). This is just a note to add this info to issue 2874.